discord/ext/commands/core.py

```diff
@@ -1210,13 +1210,17 @@ def decorator(func):
 
     return decorator
 
-def group(name=None, **attrs):
+def group(name=None, cls=None, **attrs):
     """A decorator that transforms a function into a :class:`.Group`.
 
     This is similar to the :func:`.command` decorator but creates a
-    :class:`.Group` instead of a :class:`.Command`.
+    :class:`.Group` instead of a :class:`.Command` by default.
     """
-    return command(name=name, cls=Group, **attrs)
+
+    if cls is None:
```

Comment:
```suggestion
attrs.setdefault('cls', Group)
```

Just do this instead.